### PR TITLE
Sd/window icon

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: src/glfw3.jl [hunk 1]
```diff
@@ -631,3 +632,33 @@ function standard_window_hints()
 		(AUX_BUFFERS,  0)
 	]
 end
+
+struct GLFWImage
+	width::Cint
+	height::Cint
+	pixels::Ptr{UInt8}
+	gc_ref::Any
```
Comment (jayschwa) [inline src/glfw3.jl]: What is this for?

File: src/glfw3.jl [hunk 2]
```diff
@@ -437,6 +437,7 @@ const _window_callbacks = Dict{Window, Ref{Vector{Callback}}}()
 function CreateWindow(width::Integer, height::Integer, title::AbstractString, monitor::Monitor=Monitor(C_NULL), share::Window=Window(C_NULL))
 	window = ccall( (:glfwCreateWindow, lib), Window, (Cint, Cint, Cstring, Monitor, Window), width, height, title, monitor, share)
 	callbacks = Ref{Vector{Callback}}(fill(nothing, _window_callback_num[]))
+    MakeContextCurrent(window)
```
Comment (jayschwa) [inline src/glfw3.jl]: It looks like this might be indented with spaces instead of tabs.

Comment (SimonDanisch) [reply src/glfw3.jl]: ah probably shouldn't be in there anyways!

File: src/glfw3.jl (hunk 1 above)
Comment (SimonDanisch) [reply src/glfw3.jl]: I wanted to be on the safe side for not gc'ing the input image... But after reading up a bit on those threads, I think it's save to do this in the cconvert without keeping an extra reference...

**Pull request merged**